### DISCUSSION TITLE
subdomain support for multi-tenant application

### DISCUSSION THREAD

**Discussion by eunjae-lee** [umami-software/umami#906] [Q&A] [answered]
Hello @mikecao 
I'm reading the documentation Umami, and just want to confirm my understanding.

Let's say I have a domain `example.com` and will offer subdomains for users like

```
abc.example.com
def.example.com
...
```

Having unlimited number of subdomain, does Umami fit in this scenario? In my mind,

1. When my user creates a subdomain like `abc.example.com`, 
2. then I could call `POST /api/website` to create a website at Umami database.
3. And the js from the website will send events and they will automatically attached to `abc.example.com`.

I'd appreciate if you can confirm if I understand this correctly.
Thanks!

**Comment by mikecao** [comment, marked as answer]
The domain name is only for filtering out the website from the list of referrers. What actually matters is the website id. You can place the tag on any domain you want.

**Reply by eunjae-lee** [reply]
Thanks for the explanation :)
